saritasa_invocations/alembic.py

```diff
@@ -37,13 +39,13 @@ def autogenerate(
             ),
         )
     printing.print_success("Autogenerate migrations")
-    config = context.config.get("migrations_folder", {})
-    migrations_folder = config.get(
-        "migrations_folder",
-        "db/migrations/versions",
+    config: _config.Config = context.config.get(
+        "saritasa_invocations",
+        _config.Config(),
     )
     rev_id = str(
-        len(tuple(pathlib.Path(migrations_folder).glob("*.py"))) + 1,
+        len(tuple(pathlib.Path(config.alembic_migrations_folder).glob("*.py")))
```

Comment:
Please move `pathlib.Path(config.alembic_migrations_folder).glob("*.py")` to separate var